src/amp-story-player/amp-story-player-impl.js

```diff
@@ -14,7 +14,6 @@
  * limitations under the License.
  */
 
-import * as ampCaches from '../../build-system/global-configs/caches.json';
 import * as ampToolboxCacheUrl from '@ampproject/toolbox-cache-url';
```

Comment:
This is still undefined when building. You'll need to add it to the `sources.json`

Reply:
Done.